src/main/java/hudson/plugins/gradle/Gradle.java

```diff
@@ -190,9 +185,13 @@ private boolean performTask(boolean dryRun, AbstractBuild<?, ?> build, Launcher
         //Build arguments
         ArgumentListBuilder args = new ArgumentListBuilder();
         if (useWrapper) {
-            FilePath gradleWrapperFile = getGradleWrapperFile(build, launcher, normalizedRootBuildScriptDir);
+            FilePath gradleWrapperFile = getGradleWrapperFile(build, launcher, env, normalizedRootBuildScriptDir);
+            if( !gradleWrapperFile.exists() ) {
+                listener.fatalError("Unable to find Gradle Wrapper");
```

Comment:
It would be nice if this provided more detail about where it looked for the wrapper.